app/Main.hs

```diff
@@ -9,16 +9,10 @@ import Control.Concurrent
 import GHCi.DAP.Command
 import GHCi.DAP.Type
 
-import Paths_ghci_dap (version)
-import Data.Version
-
 -- |
 --  Main
---
 main :: IO ()
 main = do
-  putStrLn $ "[DAP][INFO] start ghci-dap-" ++ showVersion version ++ "."
```

Comment:
I think it's ok just showing it to the stderr handler. (hPutStrLn stderr)

Reply:
It makes sense. Thank you for your comment. https://github.com/phoityne/ghci-dap/pull/8/commits/301cedaa39d342c40202686d14c8e90a6829331c